opensquirrel/mckay_decomposer.py

```diff
@@ -3,135 +3,115 @@
 
         lam = beta - alpha
         phi = -beta - alpha - pi
 
-        lam = normalizeAngle(lam)
-        phi = normalizeAngle(phi)
-        theta = normalizeAngle(theta)
+        lam = normalize_angle(lam)
+        phi = normalize_angle(phi)
+        theta = normalize_angle(theta)
 
         if abs(lam) > ATOL:
-            self.output.addGate("rz", qubit, lam)
+            self.output.add_gate(rz(qubit, Float(lam)))
 
-        self.output.addGate("x90", qubit)
+        self.output.add_gate(x90(qubit))
 
         if abs(theta) > ATOL:
-            self.output.addGate("rz", qubit, theta)
+            self.output.add_gate(rz(qubit, Float(theta)))
 
-        self.output.addGate("x90", qubit)
+        self.output.add_gate(x90(qubit))
 
         if abs(phi) > ATOL:
-            self.output.addGate("rz", qubit, phi)
+            self.output.add_gate(rz(qubit, Float(phi)))
 
-    def _flush(self, q):
-        if q not in self.oneQubitGates:
+    def flush(self, q):
+        if q not in self.accumulated_1q_gates:
             return
-        p = self.oneQubitGates.pop(q)
-        self._decomposeAndAdd(q, p["angle"], p["axis"])
+        p = self.accumulated_1q_gates.pop(q)
+        self.decompose_and_add(q, p["angle"], p["axis"])
 
-    def _flush_all(self):
-        while len(self.oneQubitGates) > 0:
-            self._flush(next(iter(self.oneQubitGates)))
+    def flush_all(self):
+        while len(self.accumulated_1q_gates) > 0:
+            self.flush(next(iter(self.accumulated_1q_gates)))
 
-    def _acc(self, qubit, semantic: SingleQubitAxisAngleSemantic):
-        axis, angle, phase = semantic.axis, semantic.angle, semantic.phase
+    def accumulate(self, qubit, bloch_sphere_rotation: BlochSphereRotation):
+        axis, angle, phase = bloch_sphere_rotation.axis, bloch_sphere_rotation.angle, bloch_sphere_rotation.phase
 
-        if qubit not in self.oneQubitGates:
-            self.oneQubitGates[qubit] = {"angle": angle, "axis": axis, "phase": phase}
+        if qubit not in self.accumulated_1q_gates:
+            self.accumulated_1q_gates[qubit] = {"angle": angle, "axis": axis, "phase": phase}
             return
 
-        existing = self.oneQubitGates[qubit]
-        combinedPhase = phase + existing["phase"]
+        existing = self.accumulated_1q_gates[qubit]
+        combined_phase = phase + existing["phase"]
 
         a = angle
         l = axis
         b = existing["angle"]
         m = existing["axis"]
 
-        combinedAngle = 2 * acos(cos(a / 2) * cos(b / 2) - sin(a / 2) * sin(b / 2) * np.dot(l, m))
+        combined_angle = 2 * acos(cos(a / 2) * cos(b / 2) - sin(a / 2) * sin(b / 2) * np.dot(l, m))
 
-        if abs(sin(combinedAngle / 2)) < ATOL:
-            self.oneQubitGates.pop(qubit)
+        if abs(sin(combined_angle / 2)) < ATOL:
+            self.accumulated_1q_gates.pop(qubit)
             return
 
-        combinedAxis = (
+        combined_axis = (
```

Comment:
PyCharm complains that this piece of code is unreachable. Is the above if statement always true?